src/domain/safe/entities/transfer.entity.ts

```diff
@@ -1,11 +1,10 @@
-export interface Transfer {
-  blockNumber: number;
-  executionDate: Date;
-  from: string;
-  to: string;
-  transactionHash: string;
-  transferId: string;
-}
+import { Erc20TransferSchema } from '@/domain/safe/entities/schemas/erc20-transfer.schema';
+import { Erc721TransferSchema } from '@/domain/safe/entities/schemas/erc721-transfer.schema';
+import { NativeTokenTransferSchema } from '@/domain/safe/entities/schemas/native-token-transfer.schema';
+import { TransferSchema } from '@/domain/safe/entities/schemas/transfer.schema';
+import { z } from 'zod';
+
+export type Transfer = z.infer<typeof TransferSchema>;
```

Comment:
nitpick: to move this closer to the schema.

Reply:
Moved in ee19ddaff145d1a91fca25d53853eb0c1d8b850c.